scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/RawChange.java

```diff
@@ -93,4 +95,30 @@ default boolean isDeleted(ChangeSchema.ColumnDefinition c) {
     default boolean getIsDeleted(String columnName) {
         return isDeleted(columnName);
     }
+
+    @Override
+    default Iterator<Cell> iterator() {
+        return stream().iterator();
```

Comment:
where does this `stream()` method come from/is defined? Same question to `dataStream()`.

Reply:
It is literally 11 lines below...

Reply:
Yes. I am literally blind sometimes :)